docs/pipelines.md

```diff
@@ -316,6 +317,28 @@ tasks:
 
 Note that any `task` specified in `taskSpec` will be the same version as the `Pipeline`.
 
+### Specifying Remote Tasks
+
+**([beta feature](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#beta-features))**
+
+A `taskRef` field may specify a Task in a remote location such as git.
+Support for specific types of remote will depend on the Resolvers your
+cluster's operator has installed. For more information including a tutorial, please check [resolution docs](resolution.md). The below example demonstrates referencing a Task in git:
+
+```yaml
+tasks:
+- name: "go-build"
+  taskRef:
+    resolver: git
+    params:
+    - name: url
+      value: https://github.com/tektoncd/catalog.git
+    - name: revision
+      value: $(params.gitRevision) # can use params declared at the pipeline level
```

Comment:
Nit: Wondeing if using "main" would be easier for users to use this example code for reference. 🤔

Reply:
I added this deliberately to show one could use params for the value here. I think pipeline authors should be familiar with params so hopefully it isn't too confusing

Reply:
Added a more detailed comment